modelling/src/main/java/org/axonframework/modelling/command/inspection/AnnotatedAggregateMetaModelFactory.java

```diff
@@ -91,6 +103,26 @@
                 .createModel(aggregateType);
     }
 
+    /**
+     * Shorthand to create a factory instance and inspect the model for the given {@code aggregateType} and its {@code
+     * subytpes}, using given {@code parameterResolverFactory} to resolve parameter values for annotated handlers and
+     * {@code handlerDefinition} to create concrete handlers.
+     *
+     * @param aggregateType            The class of the aggregate to create the model for
+     * @param parameterResolverFactory to resolve parameter values of annotated handlers with
+     * @param handlerDefinition        The handler definition used to create concrete handlers
+     * @param subtypes                 Subtypes of this aggregate class
+     * @param <T>                      The type of aggregate described in the model
+     * @return The model describing the structure of the aggregate
+     */
+    public static <T> AggregateModel<T> inspectAggregate(Class<T> aggregateType,
```

Comment:
Mostly out of curiosity, but why didn't you introduce a `inspectAggregate(Class, ParameterResolverFactory, List<Class>)` static method mirroring the constructors of the `AnnotatedAggregateMetaModelFactory`? Can imagine why like wouldn't be using it ourselves though.

Reply:
Just expanded to our needs. If you think it'd be valuable to add other overloads, we could.